docs/source/contribute.rst

```diff
@@ -143,13 +143,13 @@ Install the test dependencies
 
 .. code-block:: bash
 
-   python -m pip install -e ".[test]"
+   python -m pip install ".[test]"
```

Comment:
We can leave the -e if we enable the extension right?

Reply:
Maybe? I was not able to run any test on my side. I did not find a combination that works.

Reply:
Did you run `(cd tests/test_template; pip install .)` ?

Reply:
yep